TowerForge/TowerForge/ViewControllers/GameModeViewController.swift

```diff
@@ -8,7 +8,7 @@ import UIKit
 
 class GameModeViewController: UIViewController {
     @IBOutlet private var deathMatchButton: UIButton!
-    @IBOutlet private var captureTheFlagButton: UIButton!
+    @IBOutlet var captureTheFlagButton: UIButton!
```

Comment:
```suggestion
    @IBOutlet private var captureTheFlagButton: UIButton!
```

Should not expose ui element.